libraries/AC_InputManager/AC_InputManager_Heli.cpp

```diff
@@ -119,4 +122,42 @@ float AC_InputManager_Heli::get_pilot_desired_collective(int16_t control_in)
     return collective_out;
 }
 
+// parameter_check - check if input manager specific parameters are sensible
+bool AC_InputManager_Heli::parameter_check(bool display_msg) const
```

Comment:
Consider this shape instead:

https://github.com/ardupilot/ardupilot/blob/master/libraries/AP_UAVCAN/AP_UAVCAN_DNA_Server.cpp#L641